src/components/SearchBar.js

```diff
@@ -16,69 +16,87 @@ import Signals from '../api/Signals';
 declare var webkitSpeechRecognition: any; // Prevent complaints about this not existing
 
 type SearchBarProps = {
-  history: PropTypes.object.isRequired;
+  history: PropTypes.object.isRequired,
   /** If set, this will be styled to live inside a Masthead component. */
-  inMasthead: boolean;
+  inMasthead: boolean,
   /**
    * The placeholder text to display when the input field is empty and Simple
    * Query Language is selected. Defaults to “Search…”
    */
-  placeholder: string;
+  placeholder: string,
   /**
    * The placeholder text to display when the input field is empty and Advanced
    * Query Language is selected. Defaults to “Enter an advanced query…”
    */
-  placeholderAdvanced: string;
+  placeholderAdvanced: string,
   /**
    * Whether to show a toggle for simple/advanced language in the search bar.
    * Defaults to true.
    */
-  allowLanguageSelect: boolean;
+  allowLanguageSelect: boolean,
   /**
    * If set, the microphone button is displayed in the search field and the
    * user can use speech recognition to input the query terms. This functionality
    * is only available if the user’s browser is Chrome.
    */
-  allowVoice: boolean;
+  allowVoice: boolean,
   /**
    * If set, the search bar’s input field will use autocomplete via this URI.
    * Otherwise, if the configuration is available, the autoCompleteUri in the
    * configuration will be used.
    * Otherwise, the search bar will not autocomplete.
    * Note that this is relative to the baseUri field in the configuration.
    */
-  autoCompleteUri: string;
+  autoCompleteUri: string,
   /**
    * Optional. The location of the node through which to interact with Attivio.
    * Defaults to the value in the configuration.
    */
-  baseUri: string;
+  baseUri: string,
   /** The label to show on the search button. Defaults to "Go". */
-  buttonLabel: string;
+  buttonLabel: string,
   /** If set, this is the route to navigate to upon executing a search. By default, no navigation will occur when searching. */
-  route: string | null;
+  route: string | null,
+  /** Specifies if share search option should be displayed or not, false by default */
+  shareSearch: boolean,
   /**
    * If set, a new signal of this type would be added when an autocomplete item is selected.
    */
-  createAutoCompleteSignal?: boolean;
+  createAutoCompleteSignal?: boolean,
+  /**
+   * The message body for when a search is shared.
+   */
+  shareMessage: string,
```

Comment:
This needs to be optional for backwards compatibility.
```suggestion
  shareMessage?: string,
```